README.md

```diff
@@ -1 +1,163 @@
-# node-lksctp
\ No newline at end of file
+# node-lksctp
+
+## System requirements
+
+Requires libsctp of [LKSCTP](https://github.com/sctp/lksctp-tools), also known as libsctp-dev debian package.
+
+It has been tested to work with v1.0.19 and v1.0.20, but most likely also supports older versions and newer versions.
```

Comment:
It is known to fail on v1.0.18 :)

Reply:
As discussed, should be supported if compile time has right environment